content/en/apps/guides/debugging/obtaining-logs.md

```diff
@@ -22,6 +22,6 @@ The easiest way is to use `medic-logs`, a tool that comes with [`medic-conf`](ht
 medic-logs anInstance gardener
 ```
 
-See [medic-logs documentation](https://github.com/medic/medic-conf#medic-logs) for more details.
+See [medic-logs documentation](https://github.com/medic/medic-conf#user-content-fetching-logs) for more details.
```

Comment:
should probably be: https://github.com/medic/medic-conf#fetching-logs

Reply:
yes, both work! see my '[oh shucks](https://github.com/medic/cht-docs/pull/261#issuecomment-671624264)' comment about this PR wide change I made for fragment.

Reply:
cool, leaving as is then